include/portability.h

```diff
@@ -49,23 +39,6 @@ char *strerror(int errnum);
 char *strchrnul(const char *s, int c_in);
 #  endif
 
-#  ifndef HAVE_ALPHASORT
-#    include <dirent.h>
-int alphasort(const void *dirent1, const void *dirent2);
-#  endif
-
-#  ifndef HAVE_STRNLEN
-size_t strnlen(const char *s, size_t maxlen);
-#  else
-#    define USE_GNU
```

Comment:
Not sure what, if anything, `USE_GNU` did... at most, I can find descriptions of a `__USE_GNU`.